src/api/models/useTotalDataManagement.ts

```diff
@@ -18,17 +19,19 @@ export const useTotalDataManagement = () => {
         apiRequest.get('/media'),
       ]);
 
-      const reportsData = dateFormat(reportsResponse);
-      const mediaData = dateFormat(mediaResponse);
+      const reportsFormattedData = dateFormat(reportsResponse);
+      const MediaFormattedData = dateFormat(mediaResponse);
```

Comment:
카멜 케이스 사용하여 변수명을 수정해보았습니다
```suggestion
      const reportsFormattedData = dateFormat(reportsResponse);
      const mediaFormattedData = dateFormat(mediaResponse);
```

Reply:
오 놓쳤네요! 감사합니다